lib/acunetix/concerns/cleanup.rb

```diff
@@ -13,16 +13,17 @@ def cleanup_html(source)
       result.gsub!(/&lt;/, '<')
       result.gsub!(/&gt;/, '>')
 
+      result.gsub!(/<h[0-9] >(.*?)<\/h[0-9]>/) { "\n\n*#{$1.strip}*\n\n" }
```

Comment:
Is this supposed to be part of the PR?

Reply:
@aapomm that was something I came across in the sample Acunetix file (`<h4 >` tags present in the imported findings) and tried to cover in the PR description: 

> cleanup of `<h4 >` tags

Maybe it's a bit of scope creep but I was thinking of it more like "while you're in there" :)